README.md

```diff
@@ -3,6 +3,11 @@
 # Introduction
 RecipeCart an application where users can search, save, upload, and rate recipes, and ingredients for these recipes can be added to a built-in shopping list. A common problem that people face when cooking recipes is getting the right amount of ingredients based on what they already have in their selves and fridges. This aims to achieve a seamless integration of finding and cooking recipes, and shopping for (remaining) ingredients needed for these recipes. Technology-wise, the application will be a web app, where the user can login and get recipes and other information from a server.
 
+### Other documents
+- `user_manual.md` contains documentation for a user to run a RecipeCart server.
+- `developer_manual.md` contains documentation for a developer to contribute code to this repository.
```

Comment:
Good idea to link developer/user manuals in the README. I believe this was one of the optional or expected revisions.